docs/extensions/operatingsystemconfig.md

```diff
@@ -42,6 +42,18 @@ The high-level flow is as follows:
 
 1. After the VM has been provisioned the `downloader` script starts and fetches the appropriate `Secret` for its worker pool (containing the "original" user-data) and applies it.
 
+### Detailed bootstrap flow with the feature gate `BootstrapTokenForVMs` enabled
+
+If the featureFlag `BootstrapTokenForVMs` is enabled a file with the content "<<BOOTSTRAP_TOKEN>>" is added to the operatingsystem-config.
+It is passed to the worker controller (here MCM).
+
+The worker controller generates a temporary token for every new worker instance(node).
```

Comment:
```suggestion
The worker controller has to generate a temporary bootstrap token for every new worker instance(node).
```

Reply:
Actually, from the extension API itself, this description it not quite accurate. Of course, this is the recommended implementation, however, contract-wise the `Worker` controller could also do it like Gardener does it today, i.e., generating one bootstrap token for all VMs if appropriate.